api/src/main/resources/api/service_openapi.yaml

```diff
@@ -616,6 +691,38 @@ components:
         nullable: true
         $ref: "#/components/schemas/AttributeValue"
 
+    SearchEntityCountsRequest:
+      type: object
+      description: A request for counting groups of entity instances.
+      properties:
+        entityCounts:
+          $ref: "#/components/schemas/EntityCounts"
+
+    SearchEntityCountsResponse:
+      type: object
+      description: A view of entity counts for a search request.
```

Comment:
If we are supporting pagination, should we make any statement about the order of results here?

Reply:
In practice, I expect us to not need pagination for this endpoint. The current use case is for a cohort breakdown. So if there are so many groups that we need to page through them, I think it would be hard to display them in a breakdown graphic.

Regardless, I could order the results by the same group by attributes. e.g. `GROUP BY gender, race ORDER BY gender, race`. @tjennison-google would that be useful or does the order not matter to the UI (assuming everything comes back in one request)?

Reply:
I don't think the order matters and I agree that we can ignore pagination for now unless we find a need for it. The UI would need the complete results to do anything with them so I don't think pagination is useful unless we hit a response size limit or something.